ipfx/bin/get_fx_output.py

```diff
@@ -0,0 +1,71 @@
+import allensdk.internal.core.lims_utilities as lu
+import glob
+import os
+import pandas as pd
+import argparse
+
+NO_SPECIMEN = "No_specimen_in_LIMS"
+NO_OUTPUT_FILE = "No_feature_extraction_output"
+
+
+def get_fx_output_json(specimen_id):
+    """
+    Find in LIMS the full path to the json output of the feature extraction module
+    If more than one file exists chose the latest
+
+    Parameters
+    ----------
+    specimen_id
+
+    Returns
+    -------
+    file_name: string
+    """
+
+    query = """
+    select err.storage_directory, err.id
+    from specimens sp
+    join ephys_roi_results err on err.id = sp.ephys_roi_result_id
+    where sp.id = %d
+    """ % specimen_id
+
+    res = lu.query(query)
+    if res:
+        err_dir = res[0]["storage_directory"]
+
+        file_list = glob.glob(err_dir+'*EPHYS_FEATURE_EXTRACTION_*_output.json')
+        if file_list:
+            latest_file = max(file_list, key=os.path.getctime)   # get the most recent file
+            return latest_file
+        else:
+            return NO_OUTPUT_FILE
+    else:
+        return NO_SPECIMEN
+
+
+def parse_args():
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--specimen_file', type=str, required=True)
+    parser.add_argument('--specimen_output_json_file', type=str, required=True)
+    args = parser.parse_args()
+
+    return vars(args)
+
+
+def main():
```

Comment:
Maybe consider adding this to `setup.py` console scripts? https://python-packaging.readthedocs.io/en/latest/command-line-scripts.html#the-console-scripts-entry-point

Reply:
@njmei good point. If @abuchin will be using this actively then I will ad it to setup. For now, I will bookmark your link, thanks.